tests/Unit/NumericalAlgorithms/SphericalHarmonics/Test_Tags.cpp

```diff
@@ -236,13 +236,31 @@ struct SomeTag : db::SimpleTag {
   using type = SomeType;
 };
 
+void test_stf_tensor_tag() {
+  struct DummyScalar : db::SimpleTag {
```

Comment:
we have a function `TestHelpers::db::test_simple_tag<Tag>("TagName");` defined in `Helpers/DataStructures/DataBox/TestHelpers.hpp` that you should also use. It doesn't test what you do with the `static_assert`, it tests that the name matches, etc.